src/public/js/index.js

```diff
@@ -0,0 +1,28 @@
+import getTimeRemaining from './countdown';
+
+const initializeClock = (minutes) => {
+  document.getElementById('start').style.display = 'none';
+  document.getElementById('stop').style.display = 'block';
+  const endtime = Date.parse(new Date()) + (minutes * 60 * 1000); // number of miliseconds until countdown finishes
+  const clock = document.getElementById('clockdiv');
+  const timeinterval = setInterval(function () {
```

Comment:
I would prefer keeping consistency when using the arrow syntax over the function keyword

Can you change this to arrow syntax